src/vst2/Vst2PluginInstance.cpp

```diff
@@ -294,9 +294,24 @@ void Vst2PluginInstance::setProgramName(char* name)
    strcpy(programName, name);
 }
 
+//void Vst2PluginInstance::getProgramName(char* name)
+//{
+//   strcpy(name, programName);
+//}
 void Vst2PluginInstance::getProgramName(char* name)
 {
-   strcpy(name, programName);
+//   strcpy(name, programName);
+	getProgramNameIndexed(0, 0, name);
+}
+bool Vst2PluginInstance::getProgramNameIndexed (VstInt32 category, VstInt32 index, char* text)
+{//JS:
```

Comment:
I think JS indicates your initials, but we don't need that; git blame will show us you wrote it. Also JS is (funnily) not a unique initial set among the 3 people who have made C++ PRs in the last 2 weeks :)